doc/how-to/how-to-debug-failed-submissions.md

```diff
@@ -0,0 +1,94 @@
+# How to debug failed submissions
+
+This guide will explain how to troubleshoot failed application submissions. 
+
+If a live application submission fails it is important to  investigate this urgently, and aim for same-day re-submission if possible. Luckily this doesn't happen very often! 
+
+More information on how to re-submit applications can be found in [./how-to-resubmit-an-application.md](./how-to-resubmit-an-application.md).
+
+## Finding out that a submission has failed
+
+### Via `#planx-errors`
+
+Normally we find out that a submission has failed through a notification in the `#planx-errors` Slack channel, and these are usually easy to spot as they are longer than the normal errors. 
+
+They also start with `Error: Sending to <backend> failed`.
+
+E.g.
+
+```json
+> New production error in planx (error severity)
+>
+Error: Sending to BOPS v2 failed (southwark - a-5e5510n-1d-a-535510n1d): Error: Invalid DigitalPlanning payload for session a-5e5510n-1d-a-535510n1d. Errors: [   {     "instancePath": "/data/applicant/ownership/owners/0",     "schemaPath": "#/required",     "keyword": "required",     "params": {       "missingProperty": "noticeGiven"     },     "message": "must have required property 'noticeGiven'"   },   {     "instancePath": "/data/applicant/ownership/owners/0",     "schemaPath": "#/required",     "keyword": "required",     "params": {       "missingProperty": "noNoticeReason"     },     "message": "must have required property 'noNoticeReason'"   },   {     "instancePath": "/data/applicant/ownership/ow… 
+> Show more
+> Occurred in file:///api/dist/modules/send/bops/bops.js:98
+
+```
+
+### Via `#planx-notifications`
+
+There are normally three notifications per LDC submission in the `#planx-notifications` Slack channel, so a tell-tale sign of an error is if there are fewer messages. 
```

Comment:
Could we do Lawful Development Certificate (LDC) when it is first referred to then LDC after? Helps navigate the Three Letter Acronym (TLA) Forest